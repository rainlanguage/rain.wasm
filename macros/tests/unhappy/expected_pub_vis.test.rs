#[macro_use]
extern crate wasm_bindgen_utils_macros;

#[wasm_export]
fn some_fn(arg: String) -> Result<String, Error> {
    Ok(String::new())
}

#[wasm_export]
pub(crate) fn some_other_fn(arg: String) -> Result<String, Error> {
    Ok(String::new())
}

fn main() {}
