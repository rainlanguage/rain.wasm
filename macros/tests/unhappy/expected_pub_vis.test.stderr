error: expected pub visibility
 --> tests/unhappy/expected_pub_vis.test.rs:5:1
  |
5 | fn some_fn(arg: String) -> Result<String, Error> {
  | ^^

error: expected pub visibility
  --> tests/unhappy/expected_pub_vis.test.rs:10:1
   |
10 | pub(crate) fn some_other_fn(arg: String) -> Result<String, Error> {
   | ^^^^^^^^^^
